llms/mlx_lm/utils.py

```diff
@@ -8,9 +8,10 @@
 import mlx.nn as nn
 from huggingface_hub import snapshot_download
 from transformers import AutoTokenizer, PreTrainedTokenizer
+from transformers.dynamic_module_utils import resolve_trust_remote_code
```

Comment:
nit: unused import: )

Reply:
Thanks! removed in this commit: https://github.com/ml-explore/mlx-examples/pull/303/commits/d213195ee8c55861a904d92256461dea6aad5abf